README.md

```diff
@@ -2,8 +2,8 @@
 The SpatialOS Game Development Kit (GDK) for Unreal is an Unreal Engine 4 (UE4) [plugin (Unreal documentation)](https://docs.unrealengine.com/en-us/Programming/Plugins) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API.
 
 The GDK offers:
-* **Multiserver support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/13.3/shared/concepts/spatialos), the GDK enables you to use more than one Unreal game server in a single instance so that games can have more players, Actors, and gameplay systems than previously possible.
-* **An Unreal-native experience**: keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features Unreal offers out of the box, including transform synchronization, character movement, and client-side prediction.
+* **Multiserver support**: leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/concepts/spatialos), the GDK allows you to run multiple game servers in a single game instance so your Unreal-developed games can have more players, more Actors, and better gameplay systems than previously possible.
```

Comment:
This is copypasta from @Giray's update to the welcome doc (which is where this text comes from).

Reply:
I already updated this in my PR, you might need to resolve conflicts :(

Reply:
AAh, did it as I was in here removing other issue. Resolved. :-)